migrate_metadata/ckan_api.py

```diff
@@ -10,20 +14,30 @@ def __init__(self, ckan_api_url, ckan_api_key):
         self.ckan_api_key = ckan_api_key
 
     def package_list(self):
-        json_response = self._send_get_request('/api/3/action/package_list')
-        return json_response.get('result')
+        url = self.create_url('/api/3/action/package_list')
+        response = requests.get(
+            url, headers={'Authorization': self.ckan_api_key}
+            )
+        data = response.json()
+        return self.get_result(data)
 
     def package_show(self, pkg_name):
-        json_response = self._send_get_request('/api/3/action/package_show',
-                                                params={'id': pkg_name})
-        return json_response.get('result')
+        params = {'id': pkg_name}
+        url = self.create_url('/api/3/action/package_show', params=params)
+        response = requests.get(
```

Comment:
As a DRY would be nice to have a private method that sends a `get` request and already includes whatever headers are needed as well as potential error checking (e.g. check the response code).